tasks-by-users/Rehan-Khan-Engineer/06-Loading-Animation-CSS/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>JJ Loader</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="jj-ring">
```

Comment:
Give a proper class name which define the role of `div` @Rehan-Khan-Engineer

Reply:
ok i will keep this in mind in future. for now lets keep it like this.